src/app/CommandSender.cpp

```diff
@@ -145,7 +145,7 @@ CHIP_ERROR CommandSender::ProcessCommandDataElement(CommandDataElement::Parser &
             err = CHIP_NO_ERROR;
             ChipLogDetail(DataManagement, "Add Status code for empty command, cluster Id is %d", clusterId);
             // Todo: Define protocol code for StatusCode
-            AddStatusCode(0, chip::Protocols::kProtocol_Protocol_Common, 0, clusterId);
+            AddStatusCode(0, chip::Protocols::kProtocol_SecureChannel, 0, clusterId);
```

Comment:
Preexisting, but `COMMON_STATUS_SUCCESS` for the first arg?

And perhaps file a followup to have enum classes, one per protocol, for the standard-protocol status codes?

Reply:
instead of `COMMON_STATUS_SUCCESS` it should actually be `chip::Protocols::SecureChannel::GeneralStatusCode::kSuccess` (that's a mouthful...).

Will file a followup about protocol codes

Reply:
Discussed offline, general codes are already defined in this PR in src/protocols/secure_channel/Constants.h

Reply:
I was confused about what this line does.  The thing that would make sense is to have per-protocol enum classes for the protocol-specific codes, and then perhaps a version of AddStatusCode that derives the protocol from the protocol-specific code, if there is one.  But that couldn't be used in this case, since there is no protocol-specific code here.